### ISSUE TITLE
React support via Babel

### ISSUE THREAD

**Issue by joseluisq** [chinchang/web-maker#30]
Now it occurs this error:

![captura de pantalla de 2017-01-04 19-37-44](https://cloud.githubusercontent.com/assets/1700322/21654412/a678ca28-d2b5-11e6-8ce0-6196f5e1b0fc.png)

It appears like your babel version doesn't recognize JSX

Log:
```
> Uncaught TypeError: Cannot read property 'type' of undefined
    at a3 (escodegen.browser.min.js:1)
    at at (escodegen.browser.min.js:1)
    at Object.al [as generate] (escodegen.browser.min.js:1)
    at computeJs (script.js:516)
    at Object.scope.setPreviewContent (script.js:605)
    at script.js:657
a3 @ escodegen.browser.min.js:1
at @ escodegen.browser.min.js:1
al @ escodegen.browser.min.js:1
computeJs @ script.js:516
scope.setPreviewContent @ script.js:605
(anonymous) @ script.js:657

JSXTransformer.js:12705 
> You are using the in-browser JSX transformer. Be sure to precompile your JSX for production - http://facebook.github.io/react/docs/tooling-integration.html#jsx
```

**Comment by chinchang** [comment]
Will check this.

**Comment by chinchang** [comment]
@joseluisq Fixed in 2.1.0. Works now.

**Comment by joseluisq** [comment]
Awesome !

![captura de pantalla de 2017-01-18 16-19-53](https://cloud.githubusercontent.com/assets/1700322/22069802/002890c8-dd9a-11e6-9903-0089aa12666a.png)
